src/core/yaml/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 sourcemeta_library(NAMESPACE sourcemeta PROJECT core NAME yaml
   PRIVATE_HEADERS error.h
   SOURCES yaml.cc)
+sourcemeta_clang_tidy_attempt_enable(TARGET_NAME "sourcemeta_core_yaml")
```

Comment:
Can you instead do this by default on the definition of `sourcemeta_library` and `sourcemeta_executable`? Then you don't have to add it for every library